### ISSUE TITLE
resnet50-v2-7.onnx file path not found

### ISSUE THREAD

**Issue by bpickrel** [Xilinx/inference-server#86]
**Describe the bug**
The model file `resnet50-v2-7.onnx` was not found in the location expected by script `migraphx_resnet.py`  The default location used by the script was apparently not updated to reflect a change in the `proteus get` code.

**To Reproduce**
1. start docker container
2. proteus get
3. start the server
4. python3 examples/python/migraphx_resnet.py

```
waiting for server...ok.
Loading model to verify data shape...
Traceback (most recent call last):
  File "examples/python/migraphx_resnet.py", line 343, in <module>
    main(args)
  File "examples/python/migraphx_resnet.py", line 197, in main
    model = onnx.load(modelname)
  File "/usr/local/lib/python3.6/dist-packages/onnx/__init__.py", line 115, in load_model
    s = _load_bytes(f)
  File "/usr/local/lib/python3.6/dist-packages/onnx/__init__.py", line 29, in _load_bytes
    with open(cast(str, f), 'rb') as readable:
FileNotFoundError: [Errno 2] No such file or directory: '/workspace/proteus/external/artifacts/onnx/resnet50v2/resnet50-v2-7.onnx'
```

**Comment by varunsh-xilinx** [comment]
#85 is fixing this as part of the examples overhaul

**Comment by varunsh-xilinx** [comment]
Fixed in #85
